cypress/component/SideNavBarItem.cy.tsx

```diff
@@ -21,20 +21,31 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import { locator } from '@instructure/ui-test-locator'
+import React from 'react'
+import 'cypress-real-events'
 
-import { SideNavBar } from './index'
-import { SideNavBarItemLocator } from './SideNavBarItem/SideNavBarItemLocator'
+import { SideNavBarItem } from '../../packages/ui-side-nav-bar'
+import { IconAdminLine } from '../../packages/ui-icons'
+import '../support/component'
 
-export { SideNavBarItemLocator }
+describe('<SideNavBarItem/>', () => {
+  it('should show a tooltip when the nav is minimized ', async () => {
+    const onClick = cy.spy()
```

Comment:
no need to change this but is there a reason this is a spy and not just something like `() => {}`?

Reply:
This is due to the bulk rewriting of the old code stub(). It is entirely equivalent, with no particular reason. 
The only advantage is that this way the lint does not throw a warning: warning Unexpected empty method.